.pre-commit-config.yaml

```diff
@@ -158,6 +158,7 @@ repos:
         # Help installation by reducing the set of inspected botocore release.
         # There is *a lot* of them, and hatch might fetch many of them.
         - "botocore>=1.25.10"      # 1.25.10 is the current one available for RHEL9
+        - "typing-extensions>=4.9.0; python_version < '3.13'"
```

Comment:
Not related to the PR, but how does this work? Wouldn't it  install all tmt deps anyway?

Reply:
Yeah, this is a weird situation:
1. pre-commit downloads `tmt 1.35.0` and `pip install` it
2. `python3 -m tmt._pre_commit` is run
3. The code encounters `import tmt`
4. We do not have `src-layout` so instead of `import tmt` going `pre-commit`'s sources, it takes current repo's `./tmt` folder
5. Current source uses `typing-extensions`, but the dependencies were not installed because it was not defined in `tmt 1.35.0`
6. Patch the `.pre-commint-config.yaml` to manually include those dependencies as well

It's a hack that technically works, but it's soo wrong that it does.